src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Light clients
+description: Introduction to Ethereum light clients.
+lang: en
+---
+
+Running a full node is the most trustless, private, decentralized and censorship resistant way to interact with Ethereum. With a full node you keep your own copy of the blockchain that you can query instantly and you get direct access to Ethereum's peer-to-peer-network. However, running a full node requires a substantial amount of memory, storage and CPU. This can be expensive, meaning it is not feasible for everyone to run their own node. There are several solutions to this on the Ethereum roadmap, including statelessness, but they are several years away from being implemented. The answer in the near-term is to trade-off some of the benefits of running a full node for large performance improvements that allow nodes to run with very low hardware requirements. Nodes that make this trade-off are known as light nodes.
```

Comment:
> running a full node requires a substantial amount of memory, storage and CPU.

I might push back a little on the phrasing and use of "substantial" here as it seems to imply that the requirements are un-obtainably high... Of course light clients are trying to take this to another level, but much of what Ethereum strives for is to keep node requirements (even a full node today) within reason, which is emphasized in other areas of the site.

I think we could adjust the wording to remind that the full node requirements are designed to be accessible to as many people as possible, though still have hardware requirements that are not insignificant when compared to the goal of a mobile device.

A bit semantic I suppose, but that was how it read to me.

```suggestion
Running a full node is the most trustless, private, decentralized and censorship resistant way to interact with Ethereum. With a full node you keep your own copy of the blockchain that you can query instantly and you get direct access to Ethereum's peer-to-peer network. However, running a full node requires a substantial amount of memory, storage and CPU. This can be expensive, meaning it is not feasible for everyone to run their own node. There are several solutions to this on the Ethereum roadmap, including statelessness, but they are several years away from being implemented. The answer in the near-term is to trade-off some of the benefits of running a full node for large performance improvements that allow nodes to run with very low hardware requirements. Nodes that make this trade-off are known as light nodes.
```